lib/govspeak/post_processor.rb

```diff
@@ -137,6 +137,12 @@ def self.extension(title, &block)
       end
     end
 
+    extension("add tabindex=0 to tables") do |document|
+      document.css("table").map do |el|
+        el[:tabindex] = "0" unless el[:class]&.include? "js-barchart-table"
```

Comment:
Out of curiosity, where did you get `js-barchart-table` from?

Reply:
The barchart tests started failing until I added the conditional. Figured it would be easier to just not apply the tabindex to those tables rather than testing the functionality with a component we're not using :)